### PR TITLE
[RETARD]Meet the Blueshield

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Adds a new job for the game, the blueshield. His sole purpose is being a bodyguard for the heads, and he has been sent by CentComm themselves! He has a fancy stun pistol and... that's it he's just a specialized bodyguard.

Most assets ported from https://github.com/ParadiseSS13/Paradise.

## Why It's Good For The Game

Another fun job, which if i'm quite honest is just a fancy bodyguard.

**BLUESHIELDS ARE NOT STANDARD SECURITY. THEY DON'T DEAL WITH CRIMES, THEY PROTECT HEADS. THEY ARE AS GOOD AS CIVILLIANS WITH A WEAPONS LICENSE IF THERE ARE NO HEADS.**

fuckshield

## Changelog
:cl:
add: Added the Blueshield, a fancy blue security guard for the heads, and associated items.
imageadd: Blueshield item icons.
/:cl:

### CODE REVIEW

File: modular_skyrat/code/_DEFINES/jobs.dm [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #define BRIG_PHYSICIAN		(1<<14)
 #define PRISONER		(1<<15)
+#define BLUESHIELD		(1<<16)
 
-#define JOB_DISPLAY_ORDER_BRIG_PHYSICIAN 33
-#define JOB_DISPLAY_ORDER_PRISONER 34
\ No newline at end of file
+
+#define JOB_DISPLAY_ORDER_BRIG_PHYSICIAN 34
```
Comment (jjpark-kb) [inline modular_skyrat/code/_DEFINES/jobs.dm]: this needs to go back to 33 I think

File: modular_skyrat/code/_DEFINES/jobs.dm [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+#define JOB_DISPLAY_ORDER_PRISONER 36
```
Comment (jjpark-kb) [inline modular_skyrat/code/_DEFINES/jobs.dm]: this needs to go back to 34 I think

File: modular_skyrat/code/_DEFINES/jobs.dm [hunk 3]
```diff
@@ -1,5 +1,8 @@
...
+#define JOB_DISPLAY_ORDER_BLUESHIELD 34
```
Comment (jjpark-kb) [inline modular_skyrat/code/_DEFINES/jobs.dm]: this needs to be 35 I think

Comment (BobJoga) [reply modular_skyrat/code/_DEFINES/jobs.dm]: wait i fucked up hard

File: modular_skyrat/code/modules/jobs/job_types/blueshield.dm [hunk 4]
```diff
@@ -0,0 +1,54 @@
+/datum/job/blueshield
+	title = "Brig Physician"
```
Comment (jjpark-kb) [inline modular_skyrat/code/modules/jobs/job_types/blueshield.dm]: Bob?

Comment (BobJoga) [reply modular_skyrat/code/modules/jobs/job_types/blueshield.dm]: i might be going insane

File: modular_skyrat/code/modules/jobs/job_types/blueshield.dm [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+	flag = OFFICER
+	department_head = list("Captain")
+	department_flag = ENGSEC
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the Captain"
+	selection_color = "#ddddff"
+	minimal_player_age = 7
+	exp_requirements = 200 //SKYRAT CHANGE - can't have inexperience players defending a head.
```
Comment (jjpark-kb) [inline modular_skyrat/code/modules/jobs/job_types/blueshield.dm]: This is in minutes. So currently you only need to play 3.33 hours. Maybe 2400 would be good.

Comment (BobJoga) [reply modular_skyrat/code/modules/jobs/job_types/blueshield.dm]: fuck bro.... ...

**Comment by BobJoga** [comment]
ok uhhh i think i have resolved all the brain damaged, retarded shit

**Comment by BobJoga** [comment]
pls merge haha it good guys

**Comment by BobJoga** [comment]
nice map conflicts and more just fuck my shit up

**Comment by jjpark-kb** [comment]
Yea no. There are definitely some problems with what you have done with the maps. You cannot place your office ontop of shuttles. You cannot place your office ontop of the transit tubes.
You need to find a better place for it. I also dont see Box getting an office either.

**Comment by CinderWC** [comment]
Good concept, needs a **LOT** of work.

**Comment by BobJoga** [comment]
> **Stuff that needs to be changed:**
> Remove what you've done to Lambda station. That map is still in active development by Cit and probably shouldn't be played here until it is more ready.
> MetaStation changes need to be redone, it's just placed on an escape pod. Sec needs their pod.
> Rework the office on Pubby. There shouldn't be fulltile rglass giving a lovely view to... the wall of the new office.
> Rework Kilostation changes. You literally just placed it on top of transit tubes. Ya' can't do that. You also can't have it there because you're blocking external access.
> **Questions that need answering:**
> 
> 1. Why are they called 'Blueshield'? Do they have some special bluespace tech gimmick?
> 2. Why give Security yet another Detective? Aka someone who isn't security but has access.
> 3. Heads have telescopic batons, why do they need _more_ protection? (from a CE main) If anything, I'd rather not have a pseudo SecOff snooping around my department.
> 4. Why do they need an office?
> 5. If I feel like I'm in extra danger, why can't I just ask the Warden/Captain/HoS for a taser, or if I feel like I'm in a ton of danger, a laser gun?
> 6. How is the Blueballguard going to have fun if they're following me around all shift? If they aren't, then they aren't really doing their job.
> 
> I do like some of the new uniforms and such though.

1. Because they wear blue and come from the blue company, NT.
2. It's a fucking bodyguard
3. Because they're constantly targeted by traitors.
4. Where will they spawn? I guess i could put their landmark somewhere random, but if the detective has an office why wouldn't the blueshield?
6. Whoever plays blueshield is signing up for it.

About changes:
I'm not a mapper and this is all placeholder crap until Jake does it proper. I literally do not care about mapping. I don't care.

**Comment by BobJoga** [comment]
> Good concept, needs a **LOT** of work.
> 
> > combat defib and advanced health analyzer at roundstart
> > There is 2 of these blueshield medkits, so that makes 2 fucking combat defibs
> > Qvmvuqi
> 
> On top of that, the massive amounts of copypasting the Detective's office causing conflicts and breaking things such as the transit tubes is also very crazy.
> 
> For instance, this is the APC for the Blueshield Office on Box
> (There isn't even a knot under the APC so it will never charge.)
> a6Xg1y8

Removed the defibs. Only 1 medkit now, and it only has very basic stuff. About the office: I really don't care i'm not a fucking mapper.

**Comment by BobJoga** [comment]
Fuck you all. I removed the office. If shit breaks, fuck you.

**Comment by CinderWC** [comment]
What i dont get it why you just copypasted offices? why not just put a placeholder locker and blueshield spawn down somewhere in the brig instead of breaking maps just to copypaste it?

But in all seriousness, i will make a copy of this PR and check it out more thoroughly myself as i do like this

**Comment by BobJoga** [comment]
@CinderWC stole it from me >:(

**Comment by CinderWC** [comment]
epic

**Pull request closed without merging**